[12.0] Migrate and rename attachment_base_synchronize (#1756)
OCA/server-tools: attachment_queue/readme/DESCRIPTION.rst

> This module implement a queue for processing file.

Reviewer: s/implement/implements
s/file/files/

> File are stored in Odoo standard ir.attachment.

Reviewer: s/File/The files/

> This module implements a queue for processing file.
> The files are stored in Odoo standard ir.attachment.
> The attachments will be processed depending on their type.

Reviewer: ```suggestion
This module adds async processing capabilities to attachments by implementing a new model attachment.queue that wraps attachments and stores additional information so that it can be processed in an asynchronous way.
```

> 

Reviewer 2: This module adds async processing capabilities to attachments by implementing a new model attachment.queue (suggest: attachment.async.process) that wraps attachments and stores additional information so that it can be processed in an asynchronous way.

> An example of the use of this module, can be found in the module `attachment_synchronize`.

Reviewer: ```suggestion
A use case of this module can be found in the attachment_synchronize module.
```
